card_house/infrastructure/entities/db_card.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import List, Any, Dict
+
+from card_house.infrastructure.entities.card import Card
+from card_house.infrastructure.entities.card_type import CardType
+
+
+@dataclass
+class DBCard(Card):
+    type: CardType
+    content: str
+    version: str
+    creation_time: datetime
+    card_id: str
+
+    @staticmethod
+    def cards_list_to_dict(cards: List['DBCard']) -> Dict[str, Any]:
```

Comment:
Is this best practice in python to json serialize an object?
If so I'm starting to remember why I don't like this language